packages/x-date-pickers/src/MonthCalendar/MonthCalendar.types.ts

```diff
@@ -1,8 +1,27 @@
+import * as React from 'react';
 import { SxProps } from '@mui/system';
 import { Theme } from '@mui/material/styles';
 import { MonthCalendarClasses } from './monthCalendarClasses';
 import { BaseDateValidationProps, MonthValidationProps } from '../internals/models/validation';
 import { PickerValidDate, TimezoneProps } from '../models';
+import type { PickersMonthProps } from './PickersMonth';
+import { SlotComponentPropsFromProps } from '../internals/models/helpers';
+
+export interface MonthCalendarSlots {
+  /**
+   * Button displayed to render a single month in the "month" view .
+   * @default MonthCalendarButton
```

Comment:
```suggestion
   * @default PickersMonthButton
```

Reply:
With the renaming this one is no longer true :+1: